client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts

```diff
@@ -52,8 +52,7 @@ const useGroupedInvestigations = ({ invetigationsToGroup, onClose }: useGroupedI
             uniqueGroupIds: [],
             epidemiologyNumbers: []
         })
-        if (trimmedGroup.uniqueGroupIds.length === 1
-            && trimmedGroup.epidemiologyNumbers.length > 0) {
+        if (trimmedGroup.uniqueGroupIds.length === 1 && trimmedGroup.epidemiologyNumbers.length > 0) {
```

Comment:
I know you didn't write this but it should still have a comment or function that explains this.

Reply:
fixed

Reply:
```suggestion
        // Check whether there is one grouped investigation and at least one investigation without a group 
        if (trimmedGroup.uniqueGroupIds.length === 1 && trimmedGroup.epidemiologyNumbers.length > 0) {
```

Reply:
better wording